docs/config-description.md

```diff
@@ -1,452 +1,600 @@
- -   **Description**: Maximum number of requests allowed by DOS guard.
-### Key: dos_guard.sweep_interval
+- **Default value**: `20`
+- **Constraints**: The minimum value is `0`. The maximum value is `4294967295`.
+- **Description**: The maximum number of requests allowed by DOS guard.
+
+### dos_guard.sweep_interval
+
 - **Required**: True
 - **Type**: double
-- **Default value**: 1
-- **Constraints**: The value must be a positive double number
- -   **Description**: Interval in seconds for DOS guard to sweep/clear its state.
-### Key: workers
+- **Default value**: `1`
+- **Constraints**: The value must be a positive double number.
+- **Description**: Interval in seconds for DOS guard to sweep or clear its state.
+
+### workers
+
 - **Required**: True
 - **Type**: int
 - **Default value**: The number of available CPU cores.
-- **Constraints**: The minimum value is `0`. The maximum value is `4294967295`
- -   **Description**: Number of threads to process RPC requests.
-### Key: server.ip
+- **Constraints**: The minimum value is `0`. The maximum value is `4294967295`.
+- **Description**: The number of threads used to process RPC requests.
+
+### server.ip
+
 - **Required**: True
 - **Type**: string
 - **Default value**: None
-- **Constraints**: The value must be a valid IP address
- -   **Description**: IP address of the Clio HTTP server.
-### Key: server.port
+- **Constraints**: The value must be a valid IP address.
+- **Description**: The IP address of the Clio HTTP server.
+
+### server.port
+
 - **Required**: True
 - **Type**: int
 - **Default value**: None
-- **Constraints**: The minimum value is `1`. The maximum value is `65535
- -   **Description**: Port number of the Clio HTTP server.
-### Key: server.max_queue_size
+- **Constraints**: The minimum value is `1`. The maximum value is `65535`.
+- **Description**: The port number of the Clio HTTP server.
+
+### server.max_queue_size
+
 - **Required**: True
 - **Type**: int
-- **Default value**: 0
-- **Constraints**: The minimum value is `0`. The maximum value is `4294967295`
- -   **Description**: Maximum size of the server's request queue. Value of 0 is no limit.
-### Key: server.local_admin
+- **Default value**: `0`
+- **Constraints**: The minimum value is `0`. The maximum value is `4294967295`.
+- **Description**: The maximum size of the server's request queue. If set to `0`, this means there is no queue size limit.
+
+### server.local_admin
+
 - **Required**: False
 - **Type**: boolean
 - **Default value**: None
 - **Constraints**: None
- -   **Description**: Indicates if the server should run with admin privileges. Only one of local_admin or admin_password can be set.
-### Key: server.admin_password
+- **Description**: Indicates if the server should run with admin privileges. Note that this setting cannot be enabled together with [server.admin_password](#serveradmin_password), you must choose one or the other.
```

Comment:
Sounds wrong also. This is a boolean flag that indicates that requests from `127.0.0.1` are automatically granted admin permission. There is no setting to make the server run with admin privileges.